datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -591,16 +591,21 @@ SELECT c2, var_samp(CASE WHEN c12 > 0.90 THEN c12 ELSE null END) FROM aggregate_
 
 
 # csv_query_approx_median_1
-query I
+query R
+select approx_median('1');
+----
+1
+
+query R
 SELECT approx_median(c2) FROM aggregate_test_100
 ----
 3
 
 # csv_query_approx_median_2
-query I
+query R
 SELECT approx_median(c6) FROM aggregate_test_100
 ----
-1146409980542786560
```

Comment:
I guess this is due to the computation result of f64(? At least the code doesn't change the computation of `approx_median` only cast them to f64